src/nextjournal/clerk/viewer.cljc

```diff
@@ -868,6 +868,17 @@
 (def mathjax-viewer
   {:name `mathjax-viewer :render-fn 'nextjournal.clerk.render/render-mathjax :transform-fn mark-presented})
 
+(def meta-viewer
```

Comment:
I am happy to add examples to the book, @mk .

Here's one:

<img width="595" alt="image" src="https://github.com/nextjournal/clerk/assets/69635/188eb3f6-6be2-4699-87f7-78bd999bc977">